src/embed_tests/Python.EmbeddingTest.csproj

```diff
@@ -90,6 +90,7 @@
     <Compile Include="TestExample.cs" />
     <Compile Include="TestFinalizer.cs" />
     <Compile Include="TestPyAnsiString.cs" />
+	<Compile Include="TestPyBuffer.cs" />
```

Comment:
NIT: a bit off

Reply:
Hmm ive used TAB instead of spaces. I didnt know they have different length than 4 spaces on github.